package.json

```diff
@@ -85,6 +85,7 @@
   },
   "devDependencies": {
     "@commitlint/config-conventional": "^8.3.4",
+    "@rollup/plugin-json": "^4.0.1",
```

Comment:
Required since we're switching to a custom function model and ajv needs this to be bundled.